### PR TITLE
Search fields

### PR DESCRIPTION
**Pull request by xendk**
Add requesting fields to fetch from OpenPlatform. Includes updated API spec, tests, etc.

### CODE REVIEW

**Comment by xendk** [comment]
Donno what's up with phpstan, no errors locally, and on the checks page it's happy: https://github.com/reload/follow-searches/pull/40/checks?check_run_id=324740261

File: spec/follow-searches-1.1.0.yaml [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+      in: 'path'
+      description: 'Legacy user identifier of a migrated search.'
+      required: true
+      example: '29A10F616FE6CA5C6E06EBF507A9FDC5BB89F8EBCF65726BEAC61C646854E83856D3B1D46BE4696EDCFB3C9F57EEBB6941D8654BC1F4B514D6217141AEA1653C'
+      schema:
+        $ref: '#/components/schemas/LegacyUserId'
+    fields:
+      name: 'fields'
+      in: 'query'
+      description: 'Fields to request per material from OpenPlatform.'
```
Comment (kasperg) [inline spec/follow-searches-1.1.0.yaml]: This parameter is defined but never seems to be used?

Also I would appreaciate a link to the list of possible fields from OpenPlatform here.

Comment (kasperg) [reply spec/follow-searches-1.1.0.yaml]: Fixed in a7d5f09.

**Comment by kasperg** [comment]
@xendk For the record: See a984234 for the cause/fix to the PHPStan errors.

**Pull request merged**